docs/how-to/deploy_sdcore_upf_machine.md

```diff
@@ -0,0 +1,45 @@
+# Deploy SD-Core User Plane Machine
+
+This guide covers how to deploy the User Plane Function (UPF) as a machine charm.
+
+## Requirements
+
+- A host with a CPU supporting AVX2 and RDRAND instructions (Intel Haswell, AMD Excavator or equivalent)
+- [Juju][Juju] controller bootstrapped to a LXD cluster
+- A machine added to the Juju controller
```

Comment:
When I add machine to the model, it stays in `pending` state. Is this normal? So far, I've been only adding machines to models hosted on a separate Juju VM; and then it would go `idle` state.

Reply:
No, it should go to active idle.

Reply:
As discussed, that's probably because I was trying to co-locate Juju and the UPF.
I think the requirements section should state more clearly that we need 2 machines for this how-to - one for Juju and one for UPF.

Reply:
I aggree with Bart. It is better to state that Juju and UPF needs to be located on different machines.

Reply:
@gatici @Gmerold I update the branch and addressed this comment, please re-review.